modules/nodes-nodes-working-master-schedulable.adoc

```diff
@@ -13,7 +13,7 @@ default.
 
 [IMPORTANT]
 ====
-In version 4.2, the ability to create a cluster that does not have worker nodes is available to only clusters that are deployed on bare metal as a technology preview. For all other cluster types, you can set the masters to be schedulable but must retain worker nodes.
+In version {product-version}, the ability to create a cluster that does not have worker nodes is available to only clusters that are deployed on bare metal as a technology preview. For all other cluster types, you can set the masters to be schedulable but must retain worker nodes.
```

Comment:
@mburke5678 not sure if you wrote this initially, but could you double check this makes sense for 4.3 still?

Reply:
I think this note has a similar root cause as https://jira.coreos.com/browse/OSDOCS-704, and I think that @sferich888 or @abhinavdahiya could confirm.

Reply:
@sferich888 @abhinavdahiya Can you please confirm? Thanks!

Reply:
Confirm